src/screens/LoginScreen/LoginScreen.styles.ts

```diff
@@ -11,8 +11,9 @@ export default StyleSheet.create({
 		height: 50,
 		marginBottom: 15,
 		backgroundColor: 'white',
```

Comment:
Revert this file

The PR changes should be constrained to the [Trello card](https://trello.com/c/1l1Q7j24). So no need to update the login screen. Another PR request will handle the unique constraints of the login form fields.